src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -272,6 +272,12 @@ public Layout DBPassword
         [ArrayParameter(typeof(DatabaseParameterInfo), "parameter")]
         public IList<DatabaseParameterInfo> Parameters { get; } = new List<DatabaseParameterInfo>();
 
+        /// <summary>
+        /// Configures isolated transaction batch writing. If supported by the database, then it will improve insert performance.
+        /// </summary>
+        /// <docgen category='Performance Tuning Options' order='10' />
+        public System.Data.IsolationLevel? IsolationLevel { get; set; }
```

Comment:
Why is IsolationLevel fully qualified? System.Data is already in scope? What do I miss?

Reply:
It conflicts with [System.Transactions.IsolationLevel](https://docs.microsoft.com/en-us/dotnet/api/system.transactions.isolationlevel)

Reply:
Ah yes :)

Reply:
Thanks!